source/Patches/Roles/Role.cs

```diff
@@ -547,6 +586,20 @@ private static void UpdateMeeting(MeetingHud __instance)
                     }
                     else
                     {
+                        //Remove red color for impostors in meeting if anonymous impostor
+                        var playerChar = PlayerControl.AllPlayerControls.ToArray()
+                        .FirstOrDefault(x => x.PlayerId == player.TargetPlayerId);
+                        var localPlayer = PlayerControl.LocalPlayer;
+                        if (!localPlayer.Data.IsDead && localPlayer != player && player != null && playerChar.Data != null)
+                        {
+                            //Change the player name color for darker color to the medic in meeting with the GetColorType Method
+                            if (localPlayer.Is(RoleEnum.Medic) &&
```

Comment:
Since this is all specific to the Medic, can we move it into the MedicMod folder? If you're worried about conflicts with the color changing stuff here, you can use Harmony priority annotations.

Reply:
How does this interact with a Seer revealed to this player?

Reply:
If you think is better to move in medicmod folder, it's ok

since I put it in the else of role.criteria,
this should only happen if the role is not revealed.
ah but it seems to me that the seer does not have its role revealed, but only its name changed, I think in this case that at worst it should just display its name in black, but it will always see written (seer) to name side